docs/content/docs/connectors/table/formats/protobuf.md

```diff
@@ -0,0 +1,244 @@
+  uid BIGINT,
+  name STRING,
+  category_type INT,
+  bytes BINARY,
+  price DOUBLE,
+  value_map map<BIGINT, row<v1 BIGINT, v2 INT>>,
+  value_arr array<row<v1 BIGINT, v2 INT>>
+) WITH (
+ 'connector' = 'kafka',
+ 'topic' = 'user_behavior',
+ 'properties.bootstrap.servers' = 'localhost:9092',
+ 'properties.group.id' = 'testGroup',
+ 'format' = 'protobuf',
+ 'protobuf.message-class-name' = 'com.example.UserObject',
+ 'protobuf.ignore-parse-errors' = 'true'
+)
+```
+
+Format Options
+----------------
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 8%">Forwarded</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 42%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>format</h5></td>
+      <td>required</td>
+      <td>no</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what format to use, here should be <code>'protobuf'</code>.</td>
+    </tr>
+    <tr>
+      <td><h5>protobuf.message-class-name</h5></td>
+      <td>required</td>
+      <td>yes</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>The full name of a protobuf generated class. The name must match the message name in proto definition file. <code>$</code> is supported for inner class name like 'com.exmample.OuterClass$MessageClass'</td>
+    </tr>
+    <tr>
+      <td><h5>protobuf.ignore-parse-errors</h5></td>
+      <td>optional</td>
+      <td>yes</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Optional read flag to skip rows with parse errors instead of failing.</td>
+    </tr>
+    <tr>
+      <td><h5>protobuf.read-default-values</h5></td>
+      <td>optional</td>
+      <td>yes</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>
+          The option only works if then generated class's version is proto2. If the config is set to true, the format will read empty values as default values defined in proto file.
+          If the config is set to false, the format will generate null value if the data point does not exist in binary protobuf message.
+          If proto syntax is proto3, this value will be set true forcibly because proto3's standard is to use default values.
+      </td>
+    </tr>
+    <tr>
+      <td><h5>protobuf.write-null-string-literal</h5></td>
+      <td>optional</td>
+      <td>yes</td>
+      <td style="word-wrap: break-word;">""</td>
+      <td>String</td>
+      <td>
+          When serializing to protobuf data, this is the optional config to specify the string literal in protobuf's array/map in case of null values.
```

Comment:
```suggestion
          When serializing to protobuf data, this is the optional config to specify the string literal in Protobuf's array/map in case of null values.
```